mailbox/api/pom.xml

```diff
@@ -123,4 +123,17 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <reuseForks>true</reuseForks>
+                    <forkCount>4</forkCount>
```

Comment:
I would not use a fixed value here, it depends on the actual number of cpu.
You should either use `1C`. We'll have to switch to `${surefire.forkNumber}` once we have working parallel maven build (see https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#forkCount ).